ansible/roles/operator-pipeline/templates/openshift/tasks/parse-repo-changes.yml

```diff
@@ -78,3 +74,13 @@ spec:
             || fail "The PR deletes existing bundles"
         jq -e '.added_bundles|length==1' </tmp/changes.json \
             || fail "The PR affects more than one bundle"
+
+
+        jq -r '.extra_files|join(",")' </tmp/changes.json | tr -d '\n' >$(results.extra_files.path)
+        jq -r '.added_operators|join(",")' </tmp/changes.json | tr -d '\n' >$(results.added_operators.path)
+        jq -r '.modified_operators|join(",")' </tmp/changes.json | tr -d '\n' >$(results.modified_operators.path)
+        jq -r '.deleted_operators|join(",")' </tmp/changes.json | tr -d '\n' >$(results.deleted_operators.path)
+        jq -r '.added_bundles[0] | split("/")[0]' </tmp/changes.json | tr -d '\n' >$(results.added_bundle.path)
+        jq -r '.added_bundles[0] | split("/")[1]' </tmp/changes.json | tr -d '\n' >$(results.added_bundle_version.path)
```

Comment:
For consistency in the naming of the fields, I'd either remove the other outputs or align the naming (ie: using "added_operator" and "added_bundle" rather than "added_bundle" and "added_bundle_version").

Reply:
Makes sense. Fixed..